setup.py

```diff
@@ -55,6 +55,7 @@
             'License :: OSI Approved :: MIT License',
             'Programming Language :: Python',
             'Programming Language :: Python :: 2',
+            'Programming Language :: Python :: 3',
```

Comment:
Can we also add the following? (@dev-zero: is it the suggested approach to specify both python 3 and the specific version we support? Is the lower enough or we should specify all of them?)
```
Programming Language :: Python :: 2.7
Programming Language :: Python :: 3.5
Operating System :: POSIX :: Linux
Operating System :: MacOS :: MacOS X
```
(to check, we might be 3.6-only compliant, @dev-zero knows).

Also, good to check if there is anything else relevant here? 
https://pypi.org/pypi?%3Aaction=list_classifiers

Reply:
I would only declare Python 3.6 compatibility since we didn't test with 3.5.
The `Topic :: Scientific/Engineering` classifier might be a good match?

Reply:
Ok!